themes/digital.gov/layouts/partials/core/guides/guide-menu-bar.html

```diff
@@ -65,3 +65,86 @@
+        >
+          <svg
+            width="100%"
+            height="100%"
+            viewBox="0 0 21 21"
+            aria-hidden="true"
+            focusable="false"
+            class="dg-glossary__toggle-icon"
+          >
+            <path
+              d="M13.9583 2.99612C13.9583 2.25376 13.5274 1.631 12.9074 1.25803C12.2875 0.885127 11.4706 0.755653 10.6759 0.999207L7.41667 1.99802L4.15746 0.999208C3.36272 0.755654 2.54584 0.885127 1.92596 1.25803C1.30597 1.631 0.875 2.25376 0.875 2.99612V11.6299C0.875 12.5372 1.54014 13.3298 2.50921 13.6268L7.41667 15.1307L12.3241 13.6268C13.2932 13.3298 13.9583 12.5372 13.9583 11.6299V2.99612ZM11.1429 2.17693C11.4882 2.07112 11.8416 2.12807 12.107 2.28774C12.3723 2.44735 12.5417 2.7033 12.5417 2.99612V11.6299C12.5417 11.9855 12.2789 12.3198 11.8571 12.4491L8.125 13.5928V3.10179L11.1429 2.17693ZM2.29167 2.99612C2.29167 2.7033 2.46099 2.44735 2.7263 2.28774C2.99172 2.12808 3.34516 2.07112 3.69046 2.17694L6.70833 3.10179V13.5928L2.97621 12.4491C2.55441 12.3198 2.29167 11.9855 2.29167 11.6299V2.99612Z"
+            />
+          </svg>
+        </button>
+      {{- end -}}
+      <a href="{{- .Site.BaseURL -}}" title="{{- .Site.Title }} Home">
+        <picture class="dg-accordion-mobile-menu__image">
+          <source
+            srcset="{{- "img/digitalgov-logo.svg" | relURL -}}"
+            media="(min-width: 480px)"
+          />
+          <img
+            class="dg-accordion-mobile-menu__image-icon"
+            src="{{- "img/digit-100.svg" | relURL -}}"
+            alt="Digital Gov Logo"
+          />
+        </picture>
+      </a>
+    </div>
+  </div>
+  <div
+    id="a1"
+    class="usa-accordion__content dg-accordion-mobile-menu__content usa-prose"
+  >
+    <nav class="dg-accordion-mobile-menu__nav" aria-label="Secondary menu bar">
+      <ul class="dg-accordion-mobile-menu__nav-list">
+        {{- if $guideData.nav -}}
+          {{- range $guideData.nav -}}
+            {{- $item := . -}}
+            {{- $page := $.Site.GetPage $item.path -}}
+            {{- $current := false -}}
+            {{- if eq $currentPage $page.RelPermalink -}}
+              {{- $current = true -}}
+            {{- end -}}
+            {{- if $item.subnav -}}
+              {{- range $item.subnav -}}
+                {{- $subItem := . -}}
+                {{- $subPage := $.Site.GetPage $subItem.path -}}
+                {{- if eq $currentPage $subPage.RelPermalink -}}
+                  {{- $current = true -}}
+                {{- end -}}
+              {{- end -}}
+            {{- end -}}
+            <li class="dg-accordion-mobile-menu__nav-list-item">
+              <a
+                class="dg-accordion-mobile-menu__link {{ cond $current "dg-current" "" }}"
+                href="{{- $page.Permalink -}}"
+                title="{{- $item.title -}}"
+                >{{- $item.title -}}</a
+              >
+            </li>
+          {{- end -}}
+        {{- end -}}
```

Comment:
This logic for the nav list for the mobile menu is tightly coupled to the markup. I think we could move it out to another partial but need some advice on creating a more generic mobile menu component perhaps as future enhancement.